ariadne_codegen/plugins/base.py

```diff
@@ -0,0 +1,11 @@
+import ast
+
+from graphql import GraphQLSchema
+
+
+class BasePlugin:
+    def __init__(self, schema: GraphQLSchema) -> None:
+        self.schema = schema
```

Comment:
how can I pass custom configuration to a plugin?

Reply:
Hm, I haven't thought about the configuration for plugins before. I added `config_dict`(parsed `pyproject.toml`) to `Plugin` and `PluginManager`

Reply:
Is this dict a dict or typed dict that discarded unknown options?

Reply:
It's a result of `toml.load`, so basically whole `pyproject.toml` represented as dict